source/draw/gpu/GpuCanvas.ooc

```diff
@@ -72,7 +76,7 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		this _target y canvas fill()
 		this _target uv canvas fill()
 	}
-	draw: override func ~mesh (image: Image, mesh: GpuMesh) {
+	draw: override func ~mesh (image: GpuImage, mesh: GpuMesh) {
 		yuv := image as GpuYuv420Semiplanar
```

Comment:
Undefined, probably a crash somewhere. It's an ugly assumption but can be found in many places in the code. This function is currently only used in **one** place, and I expect that to be true for a long time.